client/src/App.js

```diff
@@ -1,39 +1,30 @@
-import './App.css';
 import React, { useState } from 'react';
-/*import Name from './Components/Header'*/
+import './App.css';
+import HGIB from './Components/Header';
```

Comment:
We try to make out code as descriptive as possible. Try to name your components as they are. I guess this is a header, maybe `Header`is a good name for this component ;)

Reply:
Good Point. And Done.